docs/man/sesman.ini.5.in

```diff
@@ -258,6 +277,21 @@ files on your redirected drives. This may not be approprate for all
 environents, and so you can change this value to allow other users to
 access your remote files if required.
 
+.TP
+\fBEnableFuseMount\fR=\fI[true|false]\fR
+Defaults to \fItrue\fR.
+Set to \fIfalse\fR to disable xrdp-chansrv's use of the FUSE system
+feature, even if it has been built with this feature enabled.
+.P
+.RS
+Setting this value to \fIfalse\fR will disable the following application
+features:-
+.P
+-  drive redirection
+.P
+-  cutting-and-pasting of files.
```

Comment:
Trivial point. Remove periods at the end. Otherwise put periods to both?

Reply:
Trivial's fine by me. Period removed.